src/rpc/blockchain.cpp

```diff
@@ -974,48 +1004,109 @@ static UniValue gettxoutsetinfo(const JSONRPCRequest& request)
 {
             RPCHelpMan{"gettxoutsetinfo",
                 "\nReturns statistics about the unspent transaction output set.\n"
-                "Note this call may take some time.\n",
+                "Note this call may take some time if you are not using coinstatsindex.\n",
                 {
                     {"hash_type", RPCArg::Type::STR, /* default */ "hash_serialized_2", "Which UTXO set hash should be calculated. Options: 'hash_serialized_2' (the legacy algorithm), 'none'."},
+                    {"hash_or_height", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "The block hash or height of the target height (only available with coinstatsindex)", "", {"", "string or numeric"}},
                 },
                 RPCResult{
                     RPCResult::Type::OBJ, "", "",
                     {
                         {RPCResult::Type::NUM, "height", "The current block height (index)"},
                         {RPCResult::Type::STR_HEX, "bestblock", "The hash of the block at the tip of the chain"},
```

Comment:
I think the `bestblock` help text was confusing as it was outdated so I changed it. Otherwise, I think it's ok to repeat the hash that was passed to keep consistency. Made the other change.